docs/user_interface.rst

```diff
@@ -25,284 +25,416 @@ If you do processing or acquisition which produces new data, its associated disp
+
+.. image:: graphics/inspector_line_plot.png
+    :width: 321
+    :alt: Inspector Panel with Line Plot Selected
+
+.. _Info Inspector Section:
+
+Info
+----
+The Info subsection of the inspector allows you to edit the title and description of the selected item. 
+
+.. image:: graphics/inspector_info_subsection.png
+    :width: 321
+    :alt: Inspector Info Subsection
+
+The Info subsection will only be visible if a single item is selected. 
+
+If the item selected contains multiple data items, like a layered line plot for example, changing the title and description of the item will not affect the names and descriptions of each data item; it will change the title and description for the combined display item.
+
+.. _Image Display Inspector Section:
+
+Image Display
+-------------
+Image Display allows you to force an image to be displayed as a line plot or as script instead of the default.
+
+.. image:: graphics/inspector_image_display_subsection.png
+    :width: 321
+    :alt: Inspector Image Display Subsection
+
+A converted image can be reverted back to an image by choosing Image or Default.
+
+.. _Image Data Inspector Section:
 
-The :guilabel:`Line Plot Display Layers` inspector subsection shows and allows you to edit the layers of a line plot display. In many cases, there will only be a single layer.
+Image Data
+----------
+Image Data presents several figures and settings for a selected image:
+
+.. image:: graphics/inspector_image_data_subsection.png
+    :width: 321
+    :alt: Inspector Image Data Subsection
+
+* :guilabel:`Date` - Displays the date and time a selected image was created.
+  
+* :guilabel:`Data` - Displays the dimensions of the image and the bit count.
+
+* :guilabel:`Data Range` - Displays the minimum and maximum values of the selected data.
+  
+* :guilabel:`Display Limits` - Change the minimum and maximum values currently shown in the selected data. Editing these is the same as zooming in or out on an interval on the Histogram.
+  
+* :guilabel:`Color Map` - Change the color of the data. The data range is mapped to a range of colors. Choose from a list of preset color profiles. Grayscale is the default.
+  
+* :guilabel:`Brightness` - Change the brightness of the color values on the color map. Moving the slider to the right will increase the brightness and moving the slider to the left will decrease the brightness.
+  
+* :guilabel:`Contrast` - Change the range of color values on the color map. Moving the slider to the right will increase the contrast and moving the slider to the left will decrease the contrast.
+  
+* :guilabel:`Adjustment` - Change the equalization of the selected data. Choose between Equalized, Gamma, Log, or no adjustment.
+
```

Comment:
If you choose 'Gamma', another slider control will appear to set the gamma. 1.0 is default. Range is 1/10 to 10 (I think?)